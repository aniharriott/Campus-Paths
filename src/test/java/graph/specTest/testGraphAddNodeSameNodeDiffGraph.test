# Test Graph addNode same node different graph file.
# Tests to see if the same node can be successfully
# added to two different graphs.

# Create graphs
CreateGraph graph1
CreateGraph graph2

# Add node to both
AddNode graph1 n1
AddNode graph2 n1