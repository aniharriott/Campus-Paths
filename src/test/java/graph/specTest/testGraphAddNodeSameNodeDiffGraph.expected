# Test Graph addNode same node different graph file.
# Tests to see if the same node can be successfully
# added to two different graphs.

# Create graphs
created graph graph1
created graph graph2

# Add node to both
added node n1 to graph1
added node n1 to graph2